doc/user-manual/tools/command-line-options.rst

```diff
@@ -65,6 +65,30 @@ General options
          - | Use the effect of ``2``, but also print `Loading ...`
            | when a compiled module (interface) is accessed during the type-checking.
 
+.. option:: --diagnostics-colour[=(auto|always|never)]
```

Comment:
Consider calling this option just `--colour`/`--color`, to follow conventions in:
- GNU tools, e.g.: 
  * gettext https://www.gnu.org/software/gettext/manual/html_node/The-_002d_002dcolor-option.html
  * grep
  * ls
- stack
- Idris

I guess your model was GHC:
```
$ ghc --show-options | grep color
-fdiagnostics-color=auto
-fdiagnostics-color=always
-fdiagnostics-color=never
```
But maybe GHC is the minority here.

Choices `auto/always/never` seems to be standard.

Reply:
Thanks!